specs/shipper.spec.yml

```diff
@@ -18,4 +18,9 @@ shippers:
       - kafka
       - logstash
       - redis
-    command: {}
+    command:
+      args:
+        - "-E"
+        - "logging.level=info"
+        - "-E"
+        - "logging.to_stderr=true"
```

Comment:
This is now only used in tests by the fake-shipper component (yes this is confusing there is a comment at the top about this). I will file an issue to get this cleaned up.

The one that actually gets used when the shipper is packaged with the agent is https://github.com/elastic/elastic-agent-shipper/blob/main/elastic-agent-shipper.spec.yml

Reply:
Ah! Didn't notice the comment.